test/boundaries/stagger_tests.jl

```diff
@@ -0,0 +1,69 @@
+using Attractors
+using Test
+
+@testset "Stagger Tests" begin
+
+# Dynamical system with a saddle at 0
+function F!(du, u ,p, n)
+    x,y = u
+    du[1] = x + y 
+    du[2] = x - y
+    return 
+end
```

Comment:
I love this example. This is brilliant. That's what we need for good tests, analytically resolvable situations!